CHANGELOG.md

```diff
@@ -18,7 +18,12 @@ We use [Break Versioning][breakver]. The version numbers follow a `<major>.<mino
 * Updated dependencies:
 
 ```clojure
-[metosin/malli "0.10.4"] is available but we use "0.10.1"
+[metosin/ring-swagger-ui "4.18.1"] is available but we use "4.15.5"
+[metosin/malli "0.11.0"] is available but we use "0.10.4"
+[ring/ring-core "1.10.0"] is available but we use "1.9.6"
+[manifold "0.4.0"] is available but we use "0.3.0"
+[ring "1.10.0"] is available but we use "1.9.6"
+[nubank/matcher-combinators "3.8.5"] is available but we use "3.8.4"
```

Comment:
this is a test dependency, should not be tracked.

Reply:
I'll remove it